applications/GeoMechanicsApplication/tests/cpp_tests/stub_time_loop_executor.h

```diff
@@ -12,7 +12,7 @@
 
 #pragma once
 
-#include "custom_workflows/time_loop_executor.h"
+#include "custom_workflows/time_loop_executor.hpp"
```

Comment:
I think this is not working correctly now. This stub/dummy should not derive from the 'real' time loop executor, we're missing an interface above it.

Reply:
Done